include/core/HY_Features.hpp

```diff
@@ -65,18 +66,7 @@ namespace hy_features {
          * @param network 
          * @param formulations 
          */
-        HY_Features( network::Network network, std::shared_ptr<Formulation_Manager> formulations);
```

Comment:
So far as I could tell it was unused and out of date.

Reply:
those changes are old enough now that I can't be sure, I know it was causing problems with getting layer meta data to be stored but that might have been before I updated the network class. It might work again now.

Reply:
Please add this back in if possible...